src/components/YourCompany/CorrectBusinessType.vue

```diff
@@ -260,6 +273,16 @@ export default class CorrectBusinessType extends Mixins(CommonMixin) {
     return this.getEntityType && this.getEntityType !== this.getOriginalSnapshot[0]?.business?.legalType
   }
 
+  /** Verify New Business name. */
+  private get isNewName (): boolean {
+    return this.getApprovedName && this.getApprovedName !== this.getOriginalSnapshot[0]?.business?.legalName
```

Comment:
Personally I like brackets to clarify the logic and operator precedence.

Reply:
Copy that!

Reply:
updated in next commit